plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/block-settings/index.tsx

```diff
@@ -110,6 +124,23 @@ export const ProductGalleryThumbnailsBlockSettings = ( {
 					'woocommerce'
 				) }
 			/>
+			<SelectControl
+				__next40pxDefaultSize
+				multiple={ false }
+				value={ activeThumbnailStyle }
+				options={ activeThumbnailStyleOptions }
+				label={ __( 'Active Thumbnail Style', 'woocommerce' ) }
+				onChange={ ( value ) => {
+					setAttributes( {
+						activeThumbnailStyle:
+							value as ProductGalleryThumbnailsBlockAttributes[ 'activeThumbnailStyle' ],
```

Comment:
NIT: This cast isn't safe. Potentially we should ensure that value is one of the both values.

Reply:
👍 Done in https://github.com/woocommerce/woocommerce/pull/62253/commits/2516658dd363141a51ba88103a57a8fef56704e8